ui/packages/catalog-ui-search/src/main/webapp/react-component/location/bounding-box.js

```diff
@@ -74,34 +106,149 @@ const BoundingBoxLatLon = props => {
       />
       <TextField
         label="North"
-        value={north}
-        onChange={cursor('north')}
+        value={north !== undefined ? String(north) : north}
+        onChange={value => onChangeDd('north', 'lat', value)}
+        onBlur={() => setDdError(validateGeo('lat', north))}
         type="number"
         step="any"
         min={northMin || -90}
         max={90}
         addon="°"
       />
+      {getErrorComponent(ddError)}
+    </div>
+  )
+}
+
+const BoundingBoxLatLonDms = props => {
+  const {
+    dmsSouth,
+    dmsNorth,
+    dmsWest,
+    dmsEast,
+    dmsSouthDirection,
+    dmsNorthDirection,
+    dmsWestDirection,
+    dmsEastDirection,
+    setState,
+  } = props
+  const [dmsError, setDmsError] = useState(initialErrorStateWithDefault)
+  const latitudeDirections = [Direction.North, Direction.South]
+  const longitudeDirections = [Direction.East, Direction.West]
+
+  useEffect(
+    () => {
+      console.log(props.drawing)
+      if (props.drawing) {
+        setDmsError(initialErrorStateWithDefault)
+      }
+    },
+    [props.dmsWest, props.dmsSouth, props.dmsEast, props.dmsNorth]
+  )
+
+  function validate(key, type, value) {
```

Comment:
for consistency with DD:
```suggestion
  function onChangeDms(key, type, value) {
```

Reply:
I thought it might be confusing to call it `onChange...` because it's technically being called both in the onChange and the onBlur (even though we're only specifically invoking it from the onChange). If you think it isn't confusing then I'm okay with changing it

Reply:
oh okay that makes sense, maybe we call them `validateDd` and `validateDms`?

Reply:
yeah that sounds good to me